_docs/10_general/01_introduction.md

```diff
@@ -4,10 +4,9 @@ category: general
 permalink: /
 ---
 
-In this documentation you will find most of our features explained and find instructions on how to enable certain features.
+Welcome to the Simple Analytics documentation! Here you will find detailed information on our features and how to customize them.
```

Comment:
I like the [Stripe way](https://stripe.com/docs) of welcoming: "Whether you’re a startup or a global enterprise, learn how to integrate with Stripe to accept payments and manage your business online."

```suggestion
Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.
```

Something like this?

Reply:
I like this too 👍